<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateWidget</class>
 <widget class="QWidget" name="CalibrateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Gauge Diameter:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="diameter_input">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Threshold Value:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_image_file">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Select Image File</string>
   </property>
  </widget>
  <widget class="QLabel" name="original_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>610</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: brown;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="binarized_image">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>610</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: brown;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Binary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preview_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrate_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QLabel" name="calibration_factor_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>881</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 35px; color: lightgreen; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Calibration Factor: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="diameter_unit_selection">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QSlider" name="threshold_value_slider">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="threshold_value_display">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;</string>
   </property>
   <property name="text">
    <string>127</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="is_saline_selection">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Saline</string>
   </property>
  </widget>
  <widget class="QComboBox" name="threshold_mode_selection">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="threshold_mode_tooltip_">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="saline_tooltip">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>150</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_lines">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Gauge Helper Lines</string>
   </property>
  </widget>
  <widget class="QPushButton" name="crop_image_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Crop Current Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_image_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Clear Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_binary_btn">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>130</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Save Binary Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
